src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -165,11 +165,25 @@ private function parseArgument(string $token)
         // unexpected argument
         } else {
             $all = $this->definition->getArguments();
+            $symfonyCommandName = null;
+            if (($inputArgument = $all[$key = array_key_first($all)] ?? null) && 'command' === $inputArgument->getName()) {
+                $symfonyCommandName = $this->tokens[0] ?? null;
```

Comment:
hmm, what if people use short command name ? like `d:m:m` instead of `doctrine: migration:migrate`
Just wondering what should be the output...

Reply:
Good question. The output would be: 

>   Too many arguments to "d:m:m" command, expected arguments "version".